nmmo/systems/item.py

```diff
@@ -210,6 +210,9 @@ class Armor(Equipment, ABC):
   def __init__(self, realm, level, **kwargs):
     defense = realm.config.EQUIPMENT_ARMOR_BASE_DEFENSE + \
               level*realm.config.EQUIPMENT_ARMOR_LEVEL_DEFENSE
+    # ignore quantity for Armor, so that the default quantity of 1 is used
```

Comment:
Why do we need to do this? When do you have quantity > 1?

Reply:
Other than stackable items, each item should occupy one inventory slot. But the current __init__ allows one to initiate an Armor with the quantity field of value 2 that take up one inventory space, for example. I came across this while making tests.

Reply:
@kywch There's a hard-to-reproduce bug where the assertion on line 594 of action.py fails. Can occasionally be found by test_monkey_action.py.